### DISCUSSION TITLE
fget that returns a bitfield when no flag passed

### DISCUSSION THREAD

**Discussion by sthilaid** [nesbox/TIC-80#2141] [Ideas]
What do you think of fget returning the bitfield for this tile if no flag is passed to the call (like it does on pico-8). This would be a lot more convenient to have when wanting to check multiple flags, only 1 api call is needed. 

I don't mind implementing it, it seems like it would be a trivial change.

**Comment by nesbox** [comment]
I think it's a good idea, let's do it :)
Thank you.

**Comment by joshgoebel** [comment]
So for low-level API would we pass say `0xFF` (all 8 bits set) in order to get the bitfield?  Remember, we can't pass "nothing".

**Comment by sthilaid** [comment]
figured it's easier to just do it than explain ;p

https://github.com/nesbox/TIC-80/pull/2144
